### PR TITLE
score updations for new farm and infinity vault updations [planet-finance-v2]

### PR DESCRIPTION
**Pull request by defininja**
Fixes # .

Changes proposed in this pull request:
- new farm address for aqua bnb pool
- new infinity vault address
- new aqua Lending Address

### CODE REVIEW

**Comment by defininja** [comment]
@ChaituVR - please check this pull request.

File: src/strategies/planet-finance-v2/index.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { multicall } from '../../utils';
 import { Multicaller } from '../../utils';
 import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
 
+// author: "defininja"
 export const author = 'planet-finance';
 export const version = '0.0.1';
```
Comment (ChaituVR) [inline src/strategies/planet-finance-v2/index.ts:8]:
```suggestion
export const author = 'planet-finance';
export const version = '0.0.2';
```

File: src/strategies/planet-finance-v2/index.ts [hunk 2]
```diff
@@ -4,11 +4,10 @@ import { Multicaller } from '../../utils';
 import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
 
 export const author = 'planet-finance';
```
Comment (ChaituVR) [inline src/strategies/planet-finance-v2/index.ts:6]: @defininja PR should be coming from `planet-finance`'s fork branch

Comment (defininja) [reply src/strategies/planet-finance-v2/index.ts]: @ChaituVR - Hi 

I have changed author to defininja. Please check. Thanks.

File: src/strategies/planet-finance-v2/index.ts [hunk 3]
```diff
@@ -3,12 +3,11 @@ import { multicall } from '../../utils';
 import { Multicaller } from '../../utils';
 import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
 
-export const author = 'planet-finance';
-export const version = '0.0.1';
+export const author = 'defininja';
+export const version = '0.0.3';
```
Comment (ChaituVR) [inline src/strategies/planet-finance-v2/index.ts:7]:
```suggestion
export const version = '0.0.2';
```

**Review by ChaituVR — APPROVED** [review]
✅ No open proposals

**Pull request merged**